includes/class-sensei-course.php

```diff
@@ -2005,6 +2005,37 @@ public function get_progress_statement( $course_id, $user_id ) {
 
 	}
 
+	/**
+	 * Returns a summary of course progress statistics in keyed array.
+	 *
+	 * @param int $course_id The id of the course.
+	 *
+	 * @return array $stats An array of progress stats.
+	 *               $stats['lessons_count'] The total number of lessons in the course.
+	 *               $stats['completed_lessons_count'] The total number of completed lessons of the course by the user.
+	 *               $stats['completed_lessons_percentage'] The completed lessons percentage relative to total number of lessons.
+	 */
+	public function get_progress_stats( int $course_id ): array {
```

Comment:
Good idea to create this method. Maybe we could take the opportunity to refactor the `Sensei_Course_Progress_Block` to use this method too?

Reply:
Done deal! Updated the `Sensei_Course_Progress_Block`. 💪